tests/test_io.py

```diff
@@ -3,7 +3,8 @@
 import numpy as np
 import pytest
 
-from pyam import IamDataFrame
+from pyam import IamDataFrame, read_datapackage
+from pyam.testing import assert_frame_equal
```

Comment:
would suggest

`from pyam.testing import assert_frame_equal as pyam_assert_frame_equal`

to delineate from other asserts

Reply:
renamed that function to `assert_iamframe_equal` instead